web/themes/custom/badcamp2017/sass/components/_newsletter-form.scss

```diff
@@ -0,0 +1,87 @@
+.mailchimp-signup-subscribe-form {
+
+  @include grid-column(12);
+
+  /*
```

Comment:
@bixgomez if these are going to be comment out, I'd remove them all together. I don't think they are necessary.